tests/test_config.example

```diff
@@ -26,6 +26,11 @@ XDDTEST_DEST_MOUNT=/mnt/ext4-d
 #
 XDDTEST_OUTPUT_DIR=./output
 
+#
+# Path to time command
+#
+TIME_CMD=/usr/bin/time
```

Comment:
So I actually got rid of this file all together. It was there because of the way XDD previously ran tests. Since we moved over to CMake CTest, I went ahead and updated the `README` file in the test directory explaining how to add tests.